app/controllers/photos_controller.rb

```diff
@@ -30,9 +30,7 @@ def index
           gon.preloads[:photos] = {
             count: Photo.visible(current_user, @person).count(:all)
           }
```

Comment:
I would make a helper method for this then. How about create `photos_helper.rb` and put the method in it?